README.md

```diff
@@ -52,7 +53,7 @@ fn main() {
         0x4c, 0x10, 0x00, // Jump to .algo
     ];
 
-    let mut cpu = cpu::CPU::new(Memory::new());
+    let mut cpu = cpu::CPU::new(Memory::new(), Nmos6502);
```

Comment:
Maybe we could add a `Default` impl for that?

```rust
// Initializes an Nmos6502 variant
let mut cpu = CPU::default();
```

If we don't want to add a default, we can alternatively provide

```rust
let mut cpu = CPU::nmos();
```

Just an idea.